tests/integration/targets/eos_ospfv2/tests/common/_populate.yaml

```diff
@@ -0,0 +1,53 @@
+---
+- name: Setup Ospf configs
+  arista.eos.eos_ospfv2:
```

Comment:
You really shouldn't be using the module you're testing to ensure starting state. This is why we use `eos_config` (or, where it hasn't been fixed yet, `cli_config`) in setup and teardown taksks